### PR TITLE
Move flags for stripping and scrubbing keys to agent func

### PR DESCRIPTION
**Pull request by dustmop**
### What does this PR do?

Move flags for stripping and scrubbing keys to agent func

### Motivation

These flags are referenced when loading the config, so serverless needs to have them defined.

### Describe how to test/QA your changes

Run the serverless extension. There should be no warnings about `flare_stripped_keys` or `scrubber.additional_keys` being unknown keys.

### CODE REVIEW

**Review by nhulston — APPROVED** [review]
Looks good from the Serverless side! Fixes https://github.com/DataDog/datadog-lambda-extension/issues/393.
Make sure you update the release notes.

Thank you for this work!

File: pkg/config/setup/config.go [hunk 1]
```diff
@@ -1101,6 +1087,22 @@ func agent(config pkgconfigmodel.Setup) {
...
+	config.BindEnvAndSetDefault("expected_tags_duration", time.Duration(0))
+
+	// Directory to store fleet policies
+	config.BindEnv("fleet_policies_dir")
+	config.SetDefault("fleet_layers", []string{})
+
+	// Autoscaling product
+	config.BindEnvAndSetDefault("autoscaling.workload.enabled", false)
+	config.BindEnvAndSetDefault("autoscaling.failover.enabled", false)
+	config.BindEnv("autoscaling.failover.metrics")
```
Comment (pgimalac) [inline pkg/config/setup/config.go:1104]: **🥜 nitpick:** ‏Those should probably go in a `fleet` and `autoscaling` group, doesn't seem specific to the (core) agent

File: releasenotes/notes/scrub-keys-d8faafcc427013be.yaml
Comment (pgimalac) [inline releasenotes/notes/scrub-keys-d8faafcc427013be.yaml:1]: Not really sure we need a release note for that honestly (not even sure serverless re-uses those release notes), but up to you

**Review by pgimalac — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**